workshops/geoserver/intro/doc/source/install/geoserver.rst

```diff
@@ -0,0 +1,213 @@
+* maintenance: Fixes are added to maintenance releases for the next six months.
+
+GeoServer Comunity distributions include windows and mac installers along with standalone bin download. For those that already use Tomcat a web archive is alos available.
+
+OSGeo provides a vendor neutral software foundation for Boundless, GeoSolutions and other collaborators to work on the project together. Our original company, OpenPlans, donated the GeoServer and GeoWebCache codebases to OSGeo in 2014. The OpenLayers and PostGIS projects are also hosted by OSGeo.
+
+Installation Requirements
+-------------------------
+
+In this section we will install GeoServer on your desktop computer.
+
+GeoServer, being a Java Web application, requires the following in order to function:
+
+* A Java Runtime Environment (JRE) - you may already have Java installed on your machine.
+* An an application server - our distribution includes the Jetty application server, so separate installation is not needed here.
+
+Prerequisites:
+
+* Please disable any programs on your system that use either 8080. (If this is not possible, please ask for alternative connection options.)
+* We strongly recommend you use a recent version of `Firefox <http://www.mozilla.org/en-US/firefox/new/>`__ as the browser on your host system. Using `Chrome <https://www.google.com/intl/en/chrome/browser/>`__ is acceptable, though an XML viewer extension such as `XV <https://chrome.google.com/webstore/detail/xv-%E2%80%94-xml-viewer/eeocglpgjdpaefaedpblffpeebgmgddk?hl=en>`_ will be required for some sections. Other browsers have not been tested and are not recommended.
+* Make sure you have administrative rights (Windows) or super-user privileges (Linux and OS X) on your system.
+
+.. install.geoserver.java:
+
+Installing Java
+---------------
+
+.. _Java Runtime Environment: https://java.com/en/download/manual.jsp
+
+#. Install Java on your system (for example, by downloading a `Java Runtime Environment`_ ).
+  
+   * We recommend Oracle Java Runtime Environment for Windows, Mac and Linux.
+   * On Linux OpenJDK is also supported. Check the :file:`software` folder Oracle JRE installers.
+   * We recommend running the version of Java that matches your operating system. On windows you may wish to use Java i566 (even if you use Windows 64) to allow GeoServer to run as a background service.
+#. GeoServer only requires a Java Runtime Environment (JRE). Java Development Kit (JDK) also works, including extra tools such as a compiler for development.
+
+   .. figure:: img/install_java.png
+      
+      Oracle Java Runtime Environment Installation
+
+.. install.geoserver.installer:
+
+Windows Installer
+-----------------
+
+The GeoServer installer is located in the workshop package in the :file:`software` folder.
+
+#. Visit the `http://geoserver.org/`__ website, and click on the latest stable release.
+
+    .. figure:: img/download_website.png
+       :width: 100%

```

Comment:
"width" values should not be needed. The theme should scale them appropriately.

Moreover, since I can't comment on images, many of the new ones are 200% the size of what they were, requiring the browser to scale them in a blurry way. I'd recommend keeping the images to the size they originally were.